### ISSUE TITLE
Support list view of a concept search result set

### ISSUE THREAD

**Issue by nichtich** [gbv/cocoda#410]
To simplify batch editing for instance of concepts with same label. Requires #139.

**Comment by nichtich** [comment]
Requires editable concept lists (like favorites). We start with non-editable lists (#425).

**Comment by stefandesu** [comment]
> Requires editable concept lists (like favorites). We start with non-editable lists (#425).

Should we move this to 1.4.0 then? Or should we implement this in a way that does not require full support for manual lists (i.e. the user can only create lists of search results)? There would need to be a way to delete those lists as well though.

**Comment by nichtich** [comment]
This better requires input from workflow requirements. Hard-coded searches are already supported via list configuration (just provide an API-request URL that returns the list).

**Comment by stefandesu** [comment]
> Hard-coded searches are already supported via list configuration (just provide an API-request URL that returns the list).

That's true, I forgot about that. 👍 Then let's use this if necessary for now.

**Comment by nichtich** [comment]
Requires a general discussion about the role of concept lists.
